### ISSUE TITLE
Add internal relay-command to display help and available commands

### ISSUE THREAD

**Issue by vchrizz** [geoffwhittington/meshtastic-matrix-relay#41]
Feature request for some internal relay-command like `!help` or `!commands` or similar to display all enabled plugins or available commands.
Thank you!

**Comment by geoffwhittington** [comment]
Release `0.3.21` includes this new command `!help`. Do you want to take a look?

https://github.com/geoffwhittington/meshtastic-matrix-relay/releases/tag/0.3.21

**Comment by geoffwhittington** [comment]
Thank-you for your idea submission!

**Comment by vchrizz** [comment]
but I noticed, using `!health` the bot outputs all the information now in one line.

**Comment by geoffwhittington** [comment]
Yes, I've since reverted the formatting - I had been playing with formatting support and forgot to undo. Its fixed in main and will be released in the next build
